sources/data-node/api/v2/trading_data.proto

```diff
@@ -48,6 +50,19 @@ service TradingDataService {
     // Gets the signature bundles that remove a particular validator to the multisig contract
     rpc GetERC20MultiSigSignerRemovedBundles(GetERC20MultiSigSignerRemovedBundlesRequest) returns (GetERC20MultiSigSignerRemovedBundlesResponse);
 
+    // -- Oracles --
+
+    // Get an oracle spec by ID.
+    rpc GetOracleSpecByID(GetOracleSpecByIDRequest) returns (GetOracleSpecByIDResponse);
+
+    // Get the oracle specs
+    rpc GetAllOracleSpecs(GetAllOracleSpecsRequest) returns (GetAllOracleSpecsResponse);
+
+    // Get oracle data that matched the given spec
+    rpc GetOracleDataBySpecID(GetOracleDataBySpecIDRequest) returns (GetOracleDataBySpecIDResponse);
+
+    // Get all oracle data
+    rpc GetAllOracleData(GetAllOracleDataRequest) returns (GetAllOracleDataResponse);
```

Comment:
Minor: maybe the verb `List` instead of `GetAll`? I remember some thread on slack at some point had the start of V2 new consistent API namings, and I have a feeling `List` was on there (but I can't find it) 
(cc @emilbayes )

Reply:
Alright, no problem. IWe need this to be in a `CONVENTION.md` file